.python-version

```diff
@@ -0,0 +1 @@
+3.13.2
```

Comment:
do we need this one at this level?

Also, we consistently use 3.10 atm. see https://github.com/coinbase/cdp-sdk/blob/main/python/cdp/.python-version and https://github.com/coinbase/cdp-sdk/blob/main/examples/python/.python-version